invenio_records_files/__init__.py

```diff
@@ -174,6 +181,56 @@
 >>> fileobj is None
 True
 
+Integration with Invenio REST API
+---------------------------------
+Records-Files provides views to handle files attached to a record:
+
+.. code-block:: console
+
+    # Upload a file named example.txt to the record with pid of 1
+    $ curl -X PUT http://localhost:5000/api/records/1/files/example.txt \\
```

Comment:
Apparently in older versions of Sphinx you could escape it using ``\\`` but now you can just start your docstring with ``r"""`` and it will be escaped.